### PR TITLE
Fix typo and enable multipath between only one client and only one server IP address

### PR DESCRIPTION
**Pull request by muxamilian**

### CODE REVIEW

**Review by Yanmei-Liu — APPROVED** [review]
I agree that there are scenarios which need to use multi-path with same IP address and different ports. BTW, this also help implementations test with each other when running on most devices without two or more IP addresses.

**Comment by mirjak** [comment]
Yes, it is true that multiple port number can be used and we should not that. However, the chance proposed some editorially not correct as the list is only focused on IP addresses.

**Comment by muxamilian** [comment]
My idea was that both the client and the server can only use one IP address but multiple ports. Then it's still multi-path. In what way do you mean my suggestion is not about IP addresses? I'm not sure I understood.

**Comment by mirjak** [comment]
The text before the list says

```
Each endhost may use several IP addresses to serve the connection. In
particular, the multipath extension supports the following scenarios.
```

So I think the point that you added (is correct) but doesn't find editorially in that list. We could simply add this as  sentence after the list or make a slightly larger editorial change (to this sentence above) to make sure that is clear.

**Comment by muxamilian** [comment]
@mirjak Tried to rephrase it according to your suggestion.

File: draft-ietf-quic-multipath.md [hunk 1]
```diff
@@ -153,6 +153,9 @@ particular, the multipath extension supports the following scenarios.
     several ones.
   * The client uses multiple IP addresses and the server listens on
     several ones.
+    
+The client can use one or several ports per IP address and the
+server can listen on one ore several ports per IP address.
```
Comment (qdeconinck) [inline draft-ietf-quic-multipath.md:158]:
```suggestion
    * The client uses only one IP address and the server
      listens on only one.

Note that in the last scenario, it still remains possible to have
multiple paths over the connection, given that a path is not only
defined by the IP addresses being used, but also the port numbers.
In particular, the client can use one or several ports per IP
address and the server can listen on one or several ports per IP
address.
```

**Review by qdeconinck — CHANGES_REQUESTED** [review]
In my opinion, explicitly adding a fourth scenario and explaining why multiple paths can be present there makes the idea clearer in my humble opinion.

File: draft-ietf-quic-multipath.md (hunk 1 above)
Comment (muxamilian) [reply draft-ietf-quic-multipath.md:158]: As additional context, which I forgot earlier, having multipath from the same IP but different ports is useful for congestion control algorithms that make use of fair queuing: https://github.com/muxamilian/fair-queuing-aware-congestion-control

**Pull request merged**